pkg/cmd/server/server.go

```diff
@@ -291,13 +305,50 @@ func (s *server) run() error {
 		}
 	}
 
-	if err := s.runControllers(config, backupStorageLocation); err != nil {
+	if err := s.runControllers(config, backupStorageLocation, defaultVolumeSnapshotLocations); err != nil {
 		return err
 	}
 
 	return nil
 }
 
+func (s *server) getDefaultVolumeSnapshotLocations() (map[string]*api.VolumeSnapshotLocation, error) {
```

Comment:
you should be able to add a unit test for this - create a server struct with a fake ark clientset, use the clientset to create some VSLs, set s.config.defaultVolumeSnapshotLocations, then run this func